docs/api/features/pcoords.rst

```diff
@@ -5,52 +5,54 @@ Parallel Coordinates
 
 Parallel coordinates is multi-dimensional feature visualization technique  where the vertical axis is duplicated horizontally for each feature. Instances are displayed as a single line segment drawn from each vertical axes to the location representing their value for that feature. This allows many dimensions to be visualized at once; in fact given infinite horizontal space (e.g. a scrolling window), technically an infinite number of dimensions can be displayed!
 
-Data scientists use this method to detect clusters of instances that have similar classes, and to note features that have high variance or different distributions. We can see this in action after first loading our occupancy classification dataset:
+Data scientists use this method to detect clusters of instances that have similar classes, and to note features that have high variance or different distributions. We can see this in action after first loading our occupancy classification dataset. Note that the visualization can be drawn with either the ``ParallelCoordinates`` visualizer or using the ``parallel_coordinates`` quick method:
 
-.. code:: python
+.. plot::
+
+    from yellowbrick.features import ParallelCoordinates
+    from yellowbrick.datasets import load_occupancy
 
     # Load the classification data set
-    data = load_data("occupancy")
+    X, y = load_occupancy()
 
     # Specify the features of interest and the classes of the target
     features = [
         "temperature", "relative humidity", "light", "C02", "humidity"
     ]
     classes = ["unoccupied", "occupied"]
 
-    # Extract the instances and target
-    X = data[features]
-    y = data.occupancy
-
-The visualization can be drawn with either the ``ParallelCoordinates`` visualizer or using the ``parallel_coordinates`` quick method:
-
-.. code:: python
-
-    from yellowbrick.features import ParallelCoordinates
-
     # Instantiate the visualizer
     visualizer = ParallelCoordinates(
-        classes=classes, features=features, sample=0.5, shuffle=True
+        classes=classes, features=features, sample=0.05, shuffle=True
     )
 
     # Fit and transform the data to the visualizer
     visualizer.fit_transform(X, y)
 
     # Finalize the title and axes then display the visualization
-    visualizer.poof()
-
+    visualizer.finalize()
+    visualizer.ax.grid(False)
```

Comment:
Grid lines are getting drawn for some reason so I just run ``finalize`` and turned them off

Reply:
Part of the reason we're extending the documentation with this code is so that we can get a visual confirmation of exactly what the latest version of the visualizer is doing. It could be that the grid being drawn is a bug and we need to fix it. Either way, it would probably be best to leave the code as we expect our users to run it (e.g. calling `poof()`) and then we can debug if the problem is in the visualizer or if it is something to do with the `.. plot::` directive. 

Thanks for keeping an eye out on the changes to the rendered images though, I think that's a pretty important part of going through these documentation updates.

Reply:
So I ran this code on my machine and I ended up with the following (I ran the code directly, I didn't build the documentation):

![pcoords](https://user-images.githubusercontent.com/745966/52367864-0cdffc00-2a1b-11e9-9133-977359cf97e6.png)

So this may have something to do with how the docs are configured and we can look into it.

Reply:
I ran the code also but It is different once you build the docs...

Reply:
Ok, that's good -- that means we can look into the documentation settings. If you can change the docs back to `poof()` then I can clone your PR and build the docs to see if I can spot what's going on.

Reply:
I changed them back to poof and included the images in the PR description